contracts/libraries/SnappBaseCore.sol

```diff
@@ -10,15 +10,15 @@ library SnappBaseCore {
     using Merkle for bytes32;
 
     uint public constant MAX_UINT = 2**256 - 1;
-    uint8 public constant MAX_ACCOUNT_ID = 100;
+    uint24 public constant MAX_ACCOUNT_ID = 2**24 - 2;
```

Comment:
Maybe petit comment why - 2?

Reply:
This is because -1 is the absolute max and -2 allows one extra space to make up for the shift in the Bijective mapping. In other words, we still have 2^24 -1 possible accounts, but 0 is one of them. and the "require" statement now includes equality when checking requested accountId.

Note that this detail could probably also be offloaded into the BiMap itself.

Reply:
> Note that this detail could probably also be offloaded into the BiMap itself.

I like that idea.

Also my comment was a poorly phrased request to include such a comment in the code (not just on the PR)

Reply:
I will try and implement this idea tomorrow.

Reply:
Please note that making these suggested changes will also require changes to the listener. I think maybe this would be better as a separate PR.